framework/core/hpp_instance.cpp

```diff
@@ -204,7 +204,7 @@ HPPInstance::HPPInstance(const std::string                            &applicati
 	enable_extension(VK_KHR_PORTABILITY_ENUMERATION_EXTENSION_NAME, available_instance_extensions, enabled_extensions);
 #endif
 
-#if (defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)) && defined(VKB_VALIDATION_LAYERS_GPU_ASSISTED)
+#if (defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)) && (defined(VKB_VALIDATION_LAYERS_GPU_ASSISTED) || defined(VKB_VALIDATION_LAYERS_BEST_PRACTICES))
```

Comment:
Think we might want to rethink the truth table on this one.  If you set validation best practices and it's release this would be true.

Reply:
Yeah, I'd prefer to be able to enable validation even in release builds.